projectiles/double_laser.gd

```diff
@@ -0,0 +1,33 @@
+class_name EffectDoubleLaser
+extends ProjectileEffect
+
+var Projectile : PackedScene = preload("res://units/projectile.tscn")
```

Comment:
❓ We're preloading the scene here, but also calling `load` in `_ready`. Do we need both? 🤔

Reply:
nope